### PR TITLE
Cookie choice

### PR DESCRIPTION
**Pull request by DemolaJames**
# Task:

## Description

Tracks user choice to see what % of people decline our cookie pop up.

Fixes # (issue)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Document update

## How Has This Been Tested

Please describe how to test to verify the changes. Provide instructions so we can reproduce.

## FE Checklist

- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes generate no new warnings
- [ ] Added video recordings if it is a UI change

## User Facing Checklist

- [ ] I fully understand the user problem this PR is solving
- [ ] I know who the target user is for this PR and have a deep understanding of that user
- [ ] I have tried this flow thinking from the pov of the target user for this PR
- [ ] I (or working w someone on team) have scheduled a user test for this PR (if it is a large change)

### CODE REVIEW

File: packages/frontend/src/utils/cookies.ts [hunk 1]
```diff
@@ -40,7 +41,11 @@ export const setCookie = (cookieName: string, identifier: string) => {
 
     if(consent){
         initializeAmplitude()
+        track(EVENT_NAME.COOKIE_ACCEPTED)
     }else {
         removeCookies()
+        track(EVENT_NAME.COOKIE_REJECTED)
```
Comment (KMKoushik) [inline packages/frontend/src/utils/cookies.ts:47]: Will cont work right? As we did not initialiseAmplitude

File: packages/frontend/src/components/CookiePopUp.tsx [hunk 2]
```diff
@@ -27,6 +29,9 @@ const CookiePopUp : React.FC = () => {
       <CookieConsent
         location="none"
         buttonText="Accept"
+        onAccept={() => {
+          track(EVENT_NAME.COOKIE_ACCEPTED)
+        }}
```
Comment (DemolaJames) [inline packages/frontend/src/components/CookiePopUp.tsx:34]: This is where we can track the initial  cookie rejection by calling onReject function here, but i couldnt add it cause we havent initialized amplitude... which is what i wanted to discuss with you

File: packages/frontend/src/utils/cookies.ts (hunk 1 above)
Comment (DemolaJames) [reply packages/frontend/src/utils/cookies.ts]: Changed to fathom

**Review by KMKoushik — APPROVED** [review]
LGTM

**Review by sanandnarayan — APPROVED** [review]
looks gtm. straightforward

**Comment by sanandnarayan** [comment]
@KMKoushik can merge. only alchemy test is failing

**Pull request merged**